src/auth/auth.ts

```diff
@@ -0,0 +1,76 @@
+import * as fs from 'fs';
+import * as readLine  from 'readline';
+import {google} from 'googleapis';
+
+// If modifying these scopes, delete token.json.
+const SCOPES = ['https://www.googleapis.com/auth/gmail.send'];
+// The file token.json stores the user's access and refresh tokens, and is
+// created automatically when the authorization flow completes for the first
+// time.
+const TOKEN_PATH = 'token.json';
```

Comment:
nit: I would include a whitespace line between where SCOPES ends and the comment block for TOKEN_PATH starts, also prefer /* */ comments instead